src/test/java/com/redhat/patriot/network_simulator/example/files/FileUtilsTest.java

```diff
@@ -0,0 +1,65 @@
+package com.redhat.patriot.network_simulator.example.files;
+
+import org.junit.jupiter.api.Test;
+
+import java.io.*;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+class FileUtilsTest {
+
+    @Test
+    void convertToFile() {
+
+        try {
+            File tmpDir = new File("tmpTestDir");
```

Comment:
This is not correct way how to create temporary directory.
https://docs.oracle.com/javase/8/docs/api/java/nio/file/Files.html#createTempDirectory-java.lang.String-java.nio.file.attribute.FileAttribute...-